lib/rules/web/admin_console/4.7/catalog.xyaml

```diff
@@ -69,35 +69,38 @@ create_app_from_deploy_image:
     ref: set_label
   action: click_create_button
 wait_for_catalog_loaded:
-  element:
-    selector:
+  elements:
+  - selector:
       css: .co-catalog-page
-    timeout: 20
+  - selector:
+      xpath: //div[contains(@class,'catalog-tile-pf-body')]
+  - selector:
+      xpath: //div[contains(@class,'catalog-tile-pf-header')]
```

Comment:
Adding more element checks so that we can make sure catalog item is loaded completely before going next